e3sm_to_cmip/cmor_handlers/vars/sftlf.py

```diff
@@ -14,7 +14,7 @@
 from e3sm_to_cmip import resources
 from e3sm_to_cmip._logger import _setup_logger
 from e3sm_to_cmip.mpas import write_netcdf
-from e3sm_to_cmip.util import print_message
+from e3sm_to_cmip.util import setup_cmor, print_message
```

Comment:
```suggestion
from e3sm_to_cmip.util import print_message, setup_cmor
```

Reply:
@tomvothecoder Thanks Tom for the pre-commit install information.  When you say "at the base of the e3sm_to_cmip repo directory", I assume you mean, residing in the first such directory (not the subdirectory of the same name - which is weird...).  I assume that is I every wipe and re-clone, I would need to reapply this command.  I'll add to my personal notes/documentation.